app/src/pages/TokenPage/TokenPage.tsx

```diff
@@ -25,7 +25,8 @@ export const TokenPage = (): React.ReactElement => {
   const [collectionToken, setCollectionToken] = React.useState<CollectionToken | undefined | null>(undefined);
   const [collection, setCollection] = React.useState<Collection | undefined | null>(undefined);
   const [tokenSales, setTokenSales] = React.useState<TokenTransfer[] | undefined | null>(undefined);
-
+  const [showLoadMore, setShowLoadMore] = React.useState<boolean>(true);
+  const limit = 10;
```

Comment:
give this a better name please

Reply:
in fact, give it a better name and make it a constant outside the componet, no need to make it a var

Reply:
a better name -> RECENT_SALES_PAGE_SIZE